javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,38 @@
+var width = 50;
+var height = 20;
+var area = width * height;
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+var bill = 25;
+var payment = 150;
+var change = payment - bill;
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+var quizzes = 60;
+var midterm = 80;
+var final = 90;
+var grade = (quizzes + midterm + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+var firstName = 'Danary';
+var lastName = 'Gonzalez';
+var fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof name:', typeof fullName);
+
+var pH = 24;
+var isAcidic = pH < 7;
+console.log('value of isAcidic', isAcidic);
+console.log('typeof isAcidic;', typeof isAcidic);
+
+var headCount = 320;
+var isSparta = headCount === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+var motto = fullName;
+console.log('value of motto:', fullName += ' is the GOAT');
```

Comment:
Really tiny detail here but we want to use addition assignment to concatenate the string ' is the GOAT' onto `motto`